to-do-app/src/components/ListView/index.js

```diff
@@ -22,17 +26,22 @@ class ListView extends Component {
         </ListItem>)
       
       view = <>
-        <ActionForm activeList = {this.props.activeList} saveActiveList = {this.props.saveActiveList}/>
         {items}
+        <ActionForm activeList = {this.props.activeList} saveActiveList = {this.props.saveActiveList}/>
```

Comment:
Could you remove the space: `<ActionForm activeList={this.props.activeList} saveActiveList={this.props.saveActiveList}/>`

Reply:
Sure, and not only in this place.